ctpolicy/ctpolicy.go

```diff
@@ -22,17 +22,20 @@ const (
 // CTPolicy is used to hold information about SCTs required from various
 // groupings
 type CTPolicy struct {
-	pub           pubpb.PublisherClient
-	sctLogs       loglist.List
-	infoLogs      loglist.List
-	finalLogs     loglist.List
-	stagger       time.Duration
-	log           blog.Logger
-	winnerCounter *prometheus.CounterVec
+	pub                  pubpb.PublisherClient
+	allLogs              loglist.List
```

Comment:
We wouldn't need to add `allLogs` to this struct if we emitted the relevant metrics inside of ctpolicy.New().